pypeit/core/wavecal/wvutils.py

```diff
@@ -5,10 +5,13 @@
 .. include:: ../include/links.rst
 
 """
+import os
 import numpy as np
 import os
 import numba as nb
```

Comment:
Is the number import being used? I sort of doubt it. I would comment this out.

Reply:
I presume you mean `numba` and it appears to be.